source/events.html.slim

```diff
@@ -5,18 +5,19 @@ title: Events
 section.h-lots-of-padding-top
   .body-inner-wrapper__section#about
     h1 🎉 Side Events
+    p We hope you make lots of Ruby friends at Kiwi Ruby. We're throwing some social events around the conference to facilitate this.
     - data.events.events.each do |event|
       .c-event
         h2=event.name
-        h3.c-event__details
-          span= event.time
-          span.c-event__spacer &nbsp;•&nbsp;
-          span= event.location
-        h3.c-event__details
-          span= event.payment
-          - if event.rsvp_link
-            span.c-event__spacer &nbsp;•&nbsp;
-            a(href=event.rsvp_link)=event.rsvp_text
-        - if event.description_paragraphs
-          - event.description_paragraphs.each do |paragraph|
-            p=paragraph
+        .grid
+          .col-1-2
+            h3.c-event__details= "📅 #{event.time}"
+            h3.c-event__details= "📍 #{event.location}"
+            h3.c-event__details= "💸 #{event.payment}"
+            - if event.rsvp_link
+              h3.c-event__details
+                a(href=event.rsvp_link)="📬 #{event.rsvp_text}"
+          .col-1-2
```

Comment:
...and `.col-3-4` here. They will collapse into a single column on mobile.

Reply:
Works well for the events with lots of description..
<img width="860" alt="screen shot 2017-09-05 at 1 17 14 pm" src="https://user-images.githubusercontent.com/1615322/30041241-92145c4a-923c-11e7-8eab-ab5c0c6d3bc2.png">

less well for the ones with less in the description
<img width="813" alt="screen shot 2017-09-05 at 1 17 06 pm" src="https://user-images.githubusercontent.com/1615322/30041248-a23b44c6-923c-11e7-9b52-d89e9dddeee5.png">

Could just address that by moving some of the details from the `cost` section into the description though

Reply:
Yeah, sorted, will do that 👍 
<img width="813" alt="screen shot 2017-09-05 at 1 20 55 pm" src="https://user-images.githubusercontent.com/1615322/30041304-15d26a18-923d-11e7-9a20-d4d25ac18915.png">